account_invoice_line_price_subtotal_gross/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account invoice line gross price subtotal
+=========================================
+
+This module replaces the subtotal price on account invoice line view with a new field
```

Comment:
This is misleading because as per your commit msg, you create a new field to replace the current field only on display.